src/libs/ajax.js

```diff
@@ -678,6 +678,12 @@ const Methods = signal => ({
     return res.json()
   },
 
+  template: async modifiedConfigMethod => {
+    const res = await fetchOrchestration('api/template',
```

Comment:
It looks like Rawls has an equivalent endpoint, we should prefer using that over orchestration.